package.json

```diff
@@ -52,12 +52,15 @@
     "cy:dev-oss": "source ../monitor-ci/.env && CYPRESS_dexUrl=OSS CYPRESS_baseUrl=https://$INGRESS_HOST:$PORT_HTTPS cypress open --config testFiles='{oss,shared}/**/*.*'",
     "generate": "export SHA=27c725d02cbabaf21be8bcecd43b2871840829bc && export REMOTE=https://raw.githubusercontent.com/influxdata/openapi/${SHA}/ && yarn generate-meta",
     "generate-local": "export REMOTE=../openapi/ && yarn generate-meta",
-    "generate-meta": "yarn currentApi && yarn notebooks && yarn unity && yarn flows && yarn managedFunctions && yarn annotations",
+    "generate-meta": "if [ -z \"${CLOUD_URL}\" ]; then yarn generate-meta-oss; else yarn generate-meta-cloud; fi",
+    "generate-meta-oss": "yarn oss-api && yarn notebooks && yarn unity && yarn flows && yarn managedFunctions && yarn annotations",
```

Comment:
are annotations enabled for OSS?

Reply:
according to the existing infra settings, yes!

Reply:
oss doesn't have any annotations logic at all last i knew